public/index.html

```diff
@@ -2,6 +2,8 @@
 <html lang="en">
 <head>
     <meta charset="utf-8" name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/select2@4.0.13/dist/css/select2.min.css" rel="stylesheet" />
```

Comment:
Why are these source files connected if you have already installed the select2 npm package? Use this package.